tests/test_cases/test_array_simple/test_array_simple.py

```diff
@@ -0,0 +1,79 @@
+"""
+Test getting and setting values of arrays
+"""
+import logging
+
+import cocotb
+from cocotb.binary import BinaryValue
+from cocotb.triggers import Timer
+
+tlog = logging.getLogger("cocotb.test")
+
+def _check_value(tlog, hdl, expected):
+    if hdl.value != expected:
+        raise TestFailure("{2!r}: Expected >{0}< but got >{1}<".format(expected, hdl.value, hdl))
+    else:
+        tlog.info("   Found {0!r} ({1}) with value={2}".format(hdl, hdl._type, hdl.value))
+
+
+@cocotb.test()
+def test_array_handles(dut):
+    """Test getting and setting array values using the handle of the full array.
+    """
+
+    # Set values with '<=' operator
+    dut.array_7_downto_4 <= [0xF0, 0xE0, 0xD0, 0xC0]
+    dut.array_4_to_7     <= [0xB0, 0xA0, 0x90, 0x80]
+    dut.array_3_downto_0 <= [0x70, 0x60, 0x50, 0x40]
+    dut.array_0_to_3     <= [0x30, 0x20, 0x10, 0x00]
+
+    yield Timer(1000)
```

Comment:
```suggestion
    yield Timer(1000, 'ns')
```
Let's slowly add the units in the tests too.